spec/Configuration/DotenvSpec.hs

```diff
@@ -47,26 +47,26 @@ spec = do
     it "loads the configuration options to the environment from a file" $ do
       lookupEnv "DOTENV" `shouldReturn` Nothing
 
-      void $ loadFile $ Config ["spec/fixtures/.dotenv"] [] False False
+      void $ loadFile $ Config ["spec/fixtures/.dotenv"] [] False False True
```

Comment:
@FranzGB What about creating a sample config and change it accordingly:
```hs
sampleConfig = onfig ["spec/fixtures/.dotenv"] [] False False True
-- Then, you can change the properties as needed:
loadFile $ sampleConfig { allowDuplicates = False }
loadFile $ sampleConfig { files = [] }
...
```
? I think that will make things easier to read in the test suite.

Reply:
Yes I agree that is way better thank you